src/screens/home/HomeScreen.tsx

```diff
@@ -25,6 +30,16 @@ export const HomeScreen = ({
     state: string
     value?: Record<string, unknown>
   } | null>({ state: 'default' })
+  const { addLog } = useConnectors()
+
+  const onKonnectorLog = (logObj: LogObj): void => {
```

Comment:
this logic should be written somewhere else, View files should strive to be simple

Reply:
Done